wagtail/admin/templates/wagtailadmin/shared/page_status_tag_new.html

```diff
@@ -34,7 +34,7 @@
             w-transition"
         data-tippy-offset="[0, 13]"
         aria-label="{% if is_public %}{% trans 'Visible to all. Visit the live page' %}{% else %}{% trans 'Private. Visit the live page' %}{% endif %}"
-        data-tippy-content="{% if is_public %}{% trans 'Visible to all' %}{% else %}{% trans 'Private' %}{% endif %}"
+        data-controller="{% if is_public %}{% trans 'Visible to all' %}{% else %}{% trans 'Private' %}{% endif %}"
```

Comment:
This will not work, please see main comment, the controller needs to be set to `w-tooltip` the content needs to be set with `data-w-tooltip-content-value` and the data-tippy-placement can be removed in this file as it is set to 'bottom' which is now the default with the new approach.